img/a/allmylinks.com.svg

```diff
@@ -0,0 +1,46 @@
+KlSEGCSGgMYEIsZUIIUVlLQgCcSwaFAsBDEYUfZhGwam+335oxthoPu9bmConKn7x/S7957vO8td
+zntQjaLSuKk0btWp46JNngHaCGgLtAOuAS7PPD4E7AD+DWwG9loipv8LAiqNO6AN0B/4HtACqJ2j
++3HgY2AZ8Aqw5UKJXBABlcbrAyMy7RsFDt8J/BJ4wRKxg5ecgErjrYBngDuByHlOkyLtjYmWiG27
+ZARUGr8eeBHofL4GOEtWA0MtEdta7QRUGm8CvAR0vUjgT8lyYLAlYnsKGeQKBF8DmFQN4AHuAMar
+NF5QOBZEAOgGDKwG8JCOhqFAl0IH5SUqjV8OLAZuD+nqAx8CfwW2kE7U5sBtQAegRsj414H7LRE7
+lg+uaN4EpHuB74Z0O2Zmc4DZloh9fpYBZgIPgn4C1A+Y406gN/D7fHDl5QH/27GmYEtJ77C5JIkx
+FXjKve1VZjXCt2JG1IZjzCT3ZgewDrjbVnpfhmELzYFUl8dN4iGhdiLw702k2bnAA9jquHAswPE6
+LqM9e+uEyy/XQgnI3E2gB5AIaAeRnnWr4mVh89lbXgXGz3F8EUDAcAzX7V6bCyKQ7PLTWqCxkq6Q
+REBbLJHIx2IA+Lyb8USQF5rhGKWeXmCeBhIQ6iGpTwj4HYg50XemJvPFb295wpiHY3MIiX7AredF
+oPKbjzVAGo9UEhA6PmJudM20D/K2/ikSy71PcfwCRzKAQH0c49XLKymYgNCPJG4NDn3WSVpYKPjT
+LFiMIxHihR4YfXJPkUVOdJ7SwtKnxFYB6o8Dg2uunf6HrAYojdcEGmT+3W+J2Mms/e727iK95tcJ
+0PUuRh973fsi1AMnOk12SCMltQqJ/WWSluUA3xlYCLydaQszv2Uz4Qocr4V44SaMwXl5oKLT5C6k
+t/NGARbZbXBP0T+f+kcO8L8lfXw4U7YDAywRW3vOmB94NwJ/Bq4K0LkDo5e96lXJtyoeOH7zpGKk
+cUiNQtb9l2S2Lgv4GsCYLODJ/DYm06eqFZd4G3HMD/HCtRij1bfqslo1hKReku4KDh0+lJhba92T
+fhaQDYGOAVbsmOlzrjhexLEphMR9GN/JSqC846NfQ4xF1EaQoyWR5tT+14xPuMhii7zPcMzGcTKA
+QD2McervfZXwpz0gBkq6JSRxE0KLAnDsA9YHPF+f6ZODBUtwrAzxQncc91QhcOzGiW0kjZDkAsAf
+FppVvP7pnBUES8QqgedIJ+zZsh14LtMn+/hXvMM4ZuEoCyBQhDFGg7wrAdzh9hMMaQhSs5DEXYJ4
+MzQU0qvMAGARsDXTFpFjBcrihb/heDXECx1wDACwozdMaE1602oeMO1OzHqXbHr2vVAAGcl3I8s6
+9kGvHbAUaBrQ7T2MXnak/fhxZvYMuWs7AmIl782cli+AiyEa6j0GTCP3pesExggHdJMUCYj9DZLm
+X0rwADh+g2NDYC44ejhJ7UJWnpfrvD/r8wvBcj5i87xdOF4mAjmbo0MUqXHAPGXAmkKVa7TXANEb
+uIV0leIdjL/YHO9QgV5Yg1HG6Sr32XKlk1QrwPpHJO0vCPwj3lUY83H8CsdwHCNxLMB4XmO8RoXM
+RYT9OI4EhFGxk1QZQKBIUu189S29734n0wQcvXFEzlBUA0c/jNH6sZd/OdNROxPruQikHOJAwNGh
+PlLbvBWe1M3lfrJfwEV9CJV0KIBA28ytLBeBA07S1gAPRCX6Hmw16rIwXU+07lpcdvj4hCT+1wMU
+NsExVhO9WmHzaaJ3GY6+OKIB833skFadOmbmaD2BYftbjsxZdN1w/cBIs+K6w5LlyT7llZUQsaBd
+9PsYgzTJy32dneRFMIbh6BmyI692kpZL2huSB55hUw5cN7JxFvCND0eSU0r8aNydpOjjPfvx8YOU
+FuN4EnhUk71zjtaa7DUGpuDwQuJ/H8ZyO9Dy4SKkBYK+IV5NAe/7aEUKba00XxXOb3XcJbsfc6n2
+R11l5JCrpLy2T9cbmtGkft20B3NLEtgErMDYBhiiJdAdaE/4W58liIEGsL/FyK6gJUC90NgEUoik
+iZOWosL5lLskx1ySI5EkB9wJGlxRh27tm1MzUmj1Pm85hPihTfdWZDRolcT8kBJK2qASJuEEERlR
+H2r6jpq+oyjluMyPsnvPYbZ+uS8sFy6kLcTSlUAH0GDb3CToaaQ3QhkITGASER+ivhH1LU3Ad9Ty
+IxQlHRu37eK/Bw9VB/iVGDPsiXQR+SsfN9z+wm6l66CrQs5GaRJ+msgpL9TwjZq+UeQbxX6EymNJ
+/v7Rzo0p/PcvIvi1GI/YVG/XKdxVgrTRJ/O2Ag+AlkhKBpGwjCfcWV7IhFKyxI/8affuI/dGakT6
+4ngDh38BwFM4lmEMMs/78EzMWbf1PdcMqwsMAY0Crs3aT+AjfIOk+ZwwnwrnqzyS2nE0knz+qEv+
+us/mP5YB6GdeQ4yHgYeAJgUm7GfAXMRce9w7cPbDnOeSvVcPMZldJ7gP6AVqDZRUGSPwTUqhIyed
+PqpwqaUVzl+8J1qx7Y4tr1VZQzXdc0BbjP5AT9I3wFznrHJgG7AM8Ttgi03xspVx8nvFtPvqIfVI
+fw/RlqofcRz00Y6U6YMU+qjpfxaEfjKgGZ4hGpN+XdUOoylQN/O4DPEppz4KMfbaJO+ifBRSraJZ
+nmlWAafUM+R/Ocrnwk/mQssAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjMtMDItMDlUMjI6MzI6MTAr
+MDA6MDBMpATZAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIzLTAyLTA5VDIyOjMyOjEwKzAwOjAwPfm8
+ZQAAACh0RVh0ZGF0ZTp0aW1lc3RhbXAAMjAyMy0wMi0wOVQyMjozMjoxMCswMDowMGrsnboAAAAA
+SUVORK5CYII=" />
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" id="svg69" version="1.1" viewBox="0 0 24 24"><defs id="defs33"><linearGradient id="linear-gradient" x1="6" x2="6" y1="6.67" y2="29.33" gradientTransform="translate(0 -6)" gradientUnits="userSpaceOnUse"><stop id="stop4" offset="0" stop-color="#ff4081"/><stop id="stop6" offset=".2" stop-color="#ff4081"/><stop id="stop8" offset="1" stop-color="#e91e63"/></linearGradient><linearGradient id="linear-gradient-2" x1="11.44" x2="3.88" y1="24.67" y2="29.04" gradientTransform="translate(0 -6)" gradientUnits="userSpaceOnUse"><stop id="stop11" offset="0" stop-color="#5d4037"/><stop id="stop13" offset=".1" stop-color="#5d4037" stop-opacity=".71"/><stop id="stop15" offset=".22" stop-color="#5d4037" stop-opacity=".4"/><stop id="stop17" offset=".35" stop-color="#5d4037" stop-opacity=".18"/><stop id="stop19" offset=".48" stop-color="#5d4037" stop-opacity=".04"/><stop id="stop21" offset=".6" stop-color="#5d4037" stop-opacity="0"/><stop id="stop23" offset="1" stop-color="#5d4037" stop-opacity="0"/></linearGradient><linearGradient id="linear-gradient-3" x1="17" x2="17" y1="6.67" y2="29.33" gradientTransform="translate(0 -6)" gradientUnits="userSpaceOnUse"><stop id="stop26" offset="0" stop-color="#ff4081"/><stop id="stop28" offset=".2" stop-color="#ff4081"/><stop id="stop30" offset="1" stop-color="#ff8a80"/></linearGradient><style id="style2">.cls-1{fill:#ff8a80}.cls-3{fill:#e91e63}.cls-4{fill:none}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path id="rect55" d="M0 0h24v24H0z" class="cls-4"/><path id="path57" fill="url(#linear-gradient)" d="m10.94 19.83-.67-1.16a2 2 0 0 1-.27-1 2.18 2.18 0 0 1 .3-1l-.7 1.16a3 3 0 0 1-2.6 1.5 3 3 0 0 1-3-3 3 3 0 0 1 .4-1.49l5-8.61.05-.08A3 3 0 0 1 12 4.67v-4a7 7 0 0 0-6 3.39L.94 12.83A6.9 6.9 0 0 0 0 16.31a7 7 0 0 0 7 7h.07A7.15 7.15 0 0 0 12 21.23a7.13 7.13 0 0 1-1.06-1.4Z"/><path id="path59" fill="url(#linear-gradient-2)" d="m10.94 19.83-.67-1.16a2 2 0 0 1-.27-1 2.14 2.14 0 0 1 .28-1l-.68 1.16a3 3 0 0 1-2.6 1.5 3 3 0 0 1-1.5-.41l-2 3.47a7 7 0 0 0 3.5.94h.07a7.15 7.15 0 0 0 4.93-2.1 7.13 7.13 0 0 1-1.06-1.4z"/><path id="path61" fill="url(#linear-gradient-3)" d="M17 23.33h-.08a7 7 0 0 1-6-3.5l-.67-1.16a2 2 0 0 1-.27-1 2 2 0 0 1 3.73-1l.67 1.16a3 3 0 0 0 2.62 1.5 3 3 0 0 0 3-3 3 3 0 0 0-.4-1.49l-5-8.69A2.93 2.93 0 0 0 12 4.67v-4a6.93 6.93 0 0 1 6 3.44l5 8.72a6.91 6.91 0 0 1 1 3.49 7 7 0 0 1-7 7.01z"/><circle id="circle63" cx="12" cy="11.5" r="2" fill="#ff4081"/><path id="rect65" d="M0 0h138v36H0z" class="cls-4"/></g></g></svg>
```

That SVG had an embedded Base64-encoded PNG in it. Here's an actual vector graphic.